packages/components/src/InputFile/InputFile.tsx

```diff
@@ -225,18 +230,23 @@ export function InputFile({
     } = await getUploadParams(file);
 
     const fileUpload = getFileUpload(file, key);
```

Comment:
`uploadUrl` probably should be added here since this gets spread on all the places that need it anyway. Fewer duplication and missed code